src/js/config.js

```diff
@@ -27,6 +27,7 @@ require.config({
     baseUrl: ".",
     packages: [{ name: "adapter", location: "../bower_components/spaces-adapter/src" }],
     paths: {
+        "ccLibraries": "../bower_components/ccLibraries/cc-libraries-api.min",
```

Comment:
This will be removed, and library will be loaded from `file://shared`, initial Websocket supporting CEF I had had an issue with file:// links.